### PR TITLE
Added enheder til forskelllige shops

### PR DESCRIPTION
**Pull request by DominicNr2**
titlen jaer(jæger)

### CODE REVIEW

File: src/main/java/worldofzuul/Items/Buyable.java [hunk 1]
```diff
@@ -25,4 +25,5 @@
      *
      * Gets the output in kWh for the item
      */
+
```
Comment (mathiasgredal) [inline src/main/java/worldofzuul/Items/Buyable.java]: Why is there an empty javadoc comment here

**Review by mathiasgredal — CHANGES_REQUESTED** [review]
Almost looks good

**Comment by Nolle10** [comment]
I do be fixing merge conflict 🐒🦧🦍🐵

**Review by mathiasgredal — APPROVED** [review]
LGTM

**Pull request merged**